src/devhub/entity/community/Discussions.jsx

```diff
@@ -0,0 +1,247 @@
+      console.log("error", error);
+    });
+}
+
+checkHashes();
+
+return (
+  <div className="w-100" style={{ maxWidth: "100%" }}>
+    <Container className="d-flex gap-3 m-3 pl-2">
+      <MainContent className="max-width-100">
+        <div className="d-flex flex-column gap-4">
+          {context.accountId && (
+            <div className="card p-4">
+              <Widget
+                src={"${REPL_DEVHUB}/widget/devhub.entity.community.Compose"}
+                props={{
+                  onSubmit: (v) => {
+                    Social.set(v, {
+                      force: true,
+                      onCommit: () => {
+                        getBlockHeightAndRepost();
+                      },
+                    });
+                  },
+                  communityAccountId: `discussions.${handle}.community.${REPL_DEVHUB_CONTRACT}`,
+                }}
+              />
+            </div>
+          )}
+          <div className="d-flex flex-wrap justify-content-between">
+            <Heading>Discussions</Heading>
+            <div
+              className={
+                postsExists
+                  ? "d-flex align-items-center gap-2"
+                  : " display-none"
+              }
+            >
+              <select
+                name="sort"
+                id="sort"
+                class="form-select"
+                value={sort}
+                onChange={(e) => {
+                  setSort(e.target.value);
+                }}
+              >
+                <option selected value="timedesc">
+                  Latest
+                </option>
+                <option value="recentcommentdesc">Last Commented</option>
+              </select>
+            </div>
+          </div>
+          {/* TODO: add this with the new feed {!postsExists && (
+            <div>
+              <h6>No discussions exists.</h6>
+            </div>
+          )} */}
+          <div className={postsExists && "card p-4"}>
+            {/* TODO: this feed is from https://near.org/near/widget/ComponentDetailsPage?src=mob.near/widget/ProfileTabs */}
+            <Widget
+              key="feed"
+              src="mob.near/widget/MainPage.N.Feed"
+              props={{
+                accounts: [
+                  `discussions.${handle}.community.${REPL_DEVHUB_CONTRACT}`,
+                ],
+              }}
+            />
+            {/* This is our custom feed which uses the one from near builders which should also show reposts! */}
+            <Widget
+              src="${REPL_DEVHUB}/widget/devhub.components.organism.Feed"
+              props={{
+                showFlagAccountFeature: true,
+                action: "repost",
+                filteredAccountIds: [
+                  `discussions.${handle}.community.${REPL_DEVHUB_CONTRACT}`,
+                ],
+                sort: sort,
+                setPostExists: setPostExists,
+                showFlagAccountFeature: true,
+              }}
+            />
```

Comment:
Do we need both feeds at the same time?

Reply:
Will delete the second one. I wanted to keep our custom feed to use that as soon as pogoda updates their feed to support reposts. I guess that can take a while.